src/main/java/org/solid/testharness/utils/DataRepository.java

```diff
@@ -61,6 +62,13 @@
 public class DataRepository implements Repository {
     private static final Logger logger = LoggerFactory.getLogger(DataRepository.class);
     private static final String GITHUB_LINE_ANCHOR = "#L";
+    private static final Pattern JS_ERROR = Pattern.compile(
+            "\\Rjs failed:\\R>>>>.*<<<<\\Rorg.graalvm.polyglot.PolyglotException: " +
+                    "([^\\r\\n]+)\\R" +             // exception message
+                    "(?:(Caused[^\\r\\n]+)\\R)?" +  // optional cause
+                    "(?:([^\\r\\n]+).*)?" +         // first line of stack trace
+                    "(- <js>[^\\r\\n]+).*",         // last line of stack
+            Pattern.DOTALL);
```

Comment:
OK, this I must admit I have no idea what does :-)

Reply:
Horrible isn't it. It strips down the error message. The related test in DataRepositoryTest makes it clearer.